haskell/WordCount.cabal

```diff
@@ -14,7 +14,7 @@ cabal-version:       >=1.8
 executable WordCount
   main-is:              WordCount.hs
   -- other-modules:       
-  build-depends:       base ==4.6.*, containers ==0.5.*, unordered-containers ==0.2.*, text ==1.2.*
+  build-depends:       base >=4.6.*, containers >=0.5.*, unordered-containers >=0.2.*, text >=1.2.*
```

Comment:
With latest GHC I can't use a 4.6.x base, so tried allowing later version, unfortunately fails to parse depends.  No clue :(